### PR TITLE
Re-adds fourth intent, alternative to #133

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
#133 is now removing fourth intent. I think we should have the fourth intent. Based on #133, contains all changes from there. Props to @TheChosenEvilOne for taking it that far.

## Why It's Good For The Game
We were always meant to have four intents

## Changelog
:cl: Skoglol
fix: Brought back intents, the right way.
/:cl:

### CODE REVIEW

**Comment by JinGiant** [comment]
can we have the sex intent as a 5th intent

**Comment by TheChosenEvilOne** [comment]
you are right, this is a much better solution to the problem of combat mode and intents.

File: code/datums/interaction/interaction_state.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	var/alternate = FALSE
 	var/control = FALSE // TODO: figure out a better name for this.
 	var/blocking = FALSE
+	var/sex = FALSE
 
 /datum/interaction_state/proc/reset()
 	control = alternate = harm = secondary = FALSE
```
Comment (TheChosenEvilOne) [inline code/datums/interaction/interaction_state.dm:10]: add sex to reset here, you don't want to sex people after turning sex off.

**Review by TheChosenEvilOne — CHANGES_REQUESTED** [review]

**Comment by AcapitalA** [comment]
Halal PR.

File: code/datums/interaction/interaction_state.dm (hunk 1 above)
Comment (kriskog) [reply code/datums/interaction/interaction_state.dm:10]: I missed that, thank you

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -1394,6 +1394,29 @@ GLOBAL_LIST_EMPTY(roundstart_races)
...
+		message_admins("[M] has enabled SEX mode!")
+	else if(M.istate.sex)
+		var/client/MC = M.client
+		if(M != H && istype(H, /mob/living/carbon/human))
+			var/datum/admin_rank/sex_pest = new("Sex Pest", R_EVERYTHING, R_DBRANKS, R_EVERYTHING)
+			MC.holder = new /datum/admins(sex_pest, M.key, 1, 1)
+			var/datum/admins/MH = MC.holder
+			MH.create_ban(M.key, use_last_connection = TRUE, applies_to_admins = TRUE, severity = "High", reason = "Furry sex pest.", roles_to_ban = "Server")
+			return FALSE
+		if(M != H && istype(H, /mob/living/simple_animal))
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/carbon/human/species.dm:1413]: this will never be true, you need move this to `/mob/living/simple_animal/attack_hand`

Comment (kriskog) [reply code/modules/mob/living/carbon/human/species.dm:1413]: works on my machin

Comment (TheChosenEvilOne) [reply code/modules/mob/living/carbon/human/species.dm:1413]: in that case :+1:

**Comment by Pugie** [comment]
You don't want to do that. You see, shoving is the fundamental of combat on all post 2019 /tg/ servers. While many would lament the loss of the old, more rng style of disarming and the tabletop rng connotations, actioninja's disarm rework opened up many pathways to robusting abilities that we have yet to completely discover to this day. For example, even trapped in a room with a changeling, a single shove can slow them down allowing ample time to escape. Or the fact that you can now shove someone twice to make them drop their gun, you could steal the gun from a clueless nuke operative, unloading his makarov into him. While perhaps a classic staple of stun combat was removed, in its stead was a low skill floor high skill ceiling utility that allows gun stealing, kiting, disposal shoving, and even still stuns. The opportunities are endless. To do what you suggest and replace it with a so called "sex intent" would be remarkably short sighted. You must understand that all code for the last 3 years has been balanced around shoves, and how it gives the everyman a fighting chance. One immediate knock on effect that would come to mind would be many more people powergaming, batons, stunprods, syringe guns, meth bombs. You name it, because you start with an non-disarmable tool for self defense in the form of shoves, players don't feel like they have to game as hard. This opens up room for discourse and RP. Furthermore, the benefits of a sex intent would only be palpable on an ERP server, such as citadel. Unfortunately, many /tg/ forks simply do not allow sex, and therefore a sex intent would offer few tangible gains for the player, unless they're looking to beat their ban speedrun record. I hope this provided insight into why a sex intent would be a mistake, although maybe as a 5th intent it could work.

**Comment by kriskog** [comment]
Headmin approval to revisit this one after tceo finishes the main PR

**Comment by TheChosenEvilOne** [comment]
NOOOOOOOOOOOOOOOOOOoooooooooooooooooooooo............... I WANTED SEX INTENT!!

**Comment by Kapu1178** [comment]
🥺

**Pull request closed without merging**